js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -637,3 +641,126 @@ impersonate=Impersonieren
+isBinaryAttribute=Ist Binärattribut
+isBinaryAttributeHelp=Sollte für binäre LDAP-Attribute aktiviert werden.
+passwordPolicyHintsEnabled=Passwortrichtlinien-Hinweise aktiviert
+passwordPolicyHintsEnabledHelp=Gilt nur für beschreibbares MS Active Directory (MSAD). Wenn diese Option aktiviert ist, wird beim Aktualisieren des Passworts eines MSAD-Benutzers die LDAP_SERVER_POLICY_HINTS_OID-Erweiterung verwendet, was bedeutet, dass erweiterte MSAD-Passwortrichtlinien wie "Passworthistorie" oder "minimales Passwortalter" angewendet werden. Diese Erweiterung funktioniert nur für MSAD 2008 R2 oder neuer.
+managePriorities=Prioritäten verwalten
+alwaysReadEnabledValueFromLdap="Benutzer Aktiv" Status immer aus LDAP lesen
+alwaysReadEnabledValueFromLdapHelp=Wenn diese Option aktiviert ist, wird der Aktivierungs-/Deaktivierungsstatus des Benutzers immer aus MSAD gelesen, indem der entsprechende userAccountControl Wert überprüft wird.
+alwaysReadEnabledValueFromLdapLds="Benutzer Aktiv" Status immer aus LDAP lesen
+alwaysReadEnabledValueFromLdapLdsHelp=Wenn eingeschaltet, wird der Aktivierungs-/Deaktivierungsstatus des Benutzers immer aus der MSAD-LDS gelesen, indem der Wert msDS-UserAccountDisabled überprüft wird.
+ldapDerFormatted=DER Formatiert
+ldapDerFormattedHelp=Aktivieren Sie diese Option, wenn das Zertifikat in LDAP DER-formatiert ist und nicht PEM-formatiert.
+kerberosRealm=Kerberos-Realm
+kerberosRealmHelp=Name des Kerberos-Realms, zum Beispiel: FOO.ORG
+serverPrincipal=Server-Prinzipal
+serverPrincipalHelp=Vollständiger Name des Server-Principals für den HTTP-Dienst einschließlich Server- und Domänenname. Zum Beispiel: HTTP/host.foo.org@FOO.ORG
+keyTab=Key Tab
+keyTabHelp=Speicherort der Kerberos KeyTab-Datei, die die Anmeldedaten des Server-Prinzipals enthält, zum Beispiel: /etc/krb5.keytab.
+debug=Debug
+debugHelp=Aktivieren/Deaktivieren der Debug-Protokollierung auf der Standardausgabe für das Krb5LoginModule.
+allowPasswordAuthentication=Passwort-Authentifizierung zulassen
+allowPasswordAuthenticationHelp=Aktivieren/Deaktivieren der Möglichkeit der Authentifizierung mit Benutzername/Passwort gegenüber der Kerberos-Datenbank.
+updateFirstLogin=Aktualisieren bei erster Anmeldung
+updateFirstLoginHelp=Profil bei der ersten Anmeldung aktualisieren
+requiredSettings=Erforderliche Einstellungen
+addProvider_one={{provider}} Anbieter hinzufügen
+userFedDeleteConfirmTitle=Anbieter für Benutzerföderierung löschen?
+userFedDeleteConfirm=Wenn Sie diesen Anbieter für Benutzerföderierung löschen, werden alle zugehörigen Daten entfernt.
+userFedDeletedSuccess=Der Anbieter für Benutzerföderierung wurde gelöscht.
+createUserProviderSuccess=Anbieter für Benutzerföderierung wurde erfolgreich erstellt.
+addNewProvider=Neuen Anbieter hinzufügen
+managePriorityOrder=Verwalten der Prioritätenreihenfolge
+managePriorityInfo=Die Priorität legt die Reihenfolge der Anbieter bei einer Benutzersuche fest. Sie können die Zeilenköpfe ziehen, um die Prioritäten zu ändern.
```

Comment:
Die Priorität legt die Reihenfolge der Anbieter bei einer Benutzersuche fest. Sie können die Prioritäten durch Ziehen der Zeilenschieber ändern.

Reply:
Thanks didn't know the term "Zeilenschieber" :D.